src/sentry/api/endpoints/project_transaction_names.py

```diff
@@ -51,7 +52,9 @@ def get(self, request: Request, project) -> Response:
                 "meta": {
                     "unique_transaction_names": transaction_names
                     if return_all_names
-                    else len(transaction_names)
+                    else len(transaction_names),
+                    "rules_redis": get_redis_rules(project),
+                    "rules_projectoption": get_rules(project),
```

Comment:
nit: at this point, it may be beneficial to get rid of `return_all_names` option and return everything we have there.